nailgun/entities.py

```diff
@@ -2034,6 +2034,7 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/hosts', 'server_modes': ('sat')}
         super(Host, self).__init__(server_config, **kwargs)
 
+    # pylint:disable=no-member,too-many-branches,too-many-statements
```

Comment:
Can you move this statement down into the body of the method?

```python
def create_missing(self):
    """docstring"""
    # pylint:disable=no-member,too-many-branches,too-many-statements
```

Or, if that doesn't work, try this?

```python
def create_missing(self): # pylint:disable=no-member,too-many-branches,too-many-statements
    """docstring"""
```

As is, this pylint directive applies not only to this method, but every other method that is or might be added to the `Host` class.